app/views/eligibility_interface/result/ineligible.html.erb

```diff
@@ -2,6 +2,7 @@
 
 <% content_for :page_title, "You’re not eligible" %>
 <% content_for :back_link_url, ineligible_country ? eligibility_interface_countries_path : eligibility_interface_misconduct_path %>
+<% current_date = Date.today %>
```

Comment:
Could you use `Date.current`. Using that class method on date always considers the systems time zone.